src/browser/api/window.js

```diff
@@ -1187,7 +1187,7 @@ Window.moveBy = function(identity, deltaLeft, deltaTop) {
     let left = toSafeInt(deltaLeft, 0);
     let top = toSafeInt(deltaTop, 0);
 
-    browserWindow.setBounds({
+    browserWindow.setBounds({ // no need to call clipBounds as width and height not changing
```

Comment:
Will move to separate line.
Rationale for retaining: Serves to emphasize that lack of `clipBounds` as not an oversight.